CMakeLists.txt

```diff
@@ -92,10 +92,10 @@ if(${CMAKE_VERSION} VERSION_LESS "3.16.0")
 endif()
 
 # On Linux, use the lld linker, which is twice as fast for incremental debug builds
-if (NOT APPLE)
-    set(CMAKE_SHARED_LINKER_FLAGS "-fuse-ld=lld")
-    set(CMAKE_EXE_LINKER_FLAGS "-fuse-ld=lld")
-endif()
+# if (NOT APPLE)
+#     set(CMAKE_SHARED_LINKER_FLAGS "-fuse-ld=lld")
+#     set(CMAKE_EXE_LINKER_FLAGS "-fuse-ld=lld")
+# endif()
```

Comment:
?
mkrs already recommended a lot of changes in PR 2025. We will try to avoid duplication.